### ISSUE TITLE
Can't make semicolons: false rule work

### ISSUE THREAD

**Issue by tmilar** [prettier/tslint-config-prettier#374]
I've tried many combinations but I can't make a no semicolons rule be enforced.

These are my "devDependencies":
```
    "prettier": "1.19.1",
    "tslint": "5.20.1",
    "tslint-config-prettier": "1.18.0",
    "tslint-plugin-prettier": "2.1.0"
```
This is my `tslint.json` : 

```json
{
  "defaultSeverity": "error",
  "extends": [
    "tslint:recommended",
    "tslint-config-prettier"
  ],
  "jsRules": {},
  "rules": {
    "semi": ["error", "never"],
    "prettier": true,
    "no-console": [
      false
    ]
  },
  "rulesDirectory": [
    "tslint-plugin-prettier"
  ]
}
```

I've tried many combinations inside the `"rules"` object, such as:

* `"semicolon": [true, "never"],` // works for tslint, but then conflicts with prettier 
* `"semi": [true, "never"],` // does nothing
* `"semi": false,` // does nothing
* `"prettier.semi": false,` // does nothing
* `"prettier/semi": false,` // does nothing
* `"prettier: { "semi": false },` // (instead of just `"prettier": true` , but still nothing

Also tried many combinations by defining a `.prettierrc` file with this rule, but to no avail.
Is there anything I'm missing here?

Thanks!

**Comment by ikatyang** [comment]
The _no semicolon_ rule should be enforced by Prettier via 

```js
// tslint.json
{ 
  "extends": ["tslint:recommended", "tslint-config-prettier"],
  "rulesDirectory": ["tslint-plugin-prettier"],
  "rules": { "prettier": [true, { "semi": false }] }
}
```

or

```js
// .prettierrc
{ "semi": false }

// tslint.json
{ 
  "extends": ["tslint:recommended", "tslint-config-prettier"],
  "rulesDirectory": ["tslint-plugin-prettier"],
  "rules": { "prettier": true }
}
```

The TSLint `semicolon` rule may be conflict with Prettier so we simply disable it via `tslint-config-prettier`.

**Comment by tmilar** [comment]
Thanks, I had to do a combination of both for it to work.

If I only changed tslint.json, like your first suggestion, it would stop showing errors when no semicolons are used, but the Prettier Reformat action would put them back again (current IDE: Webstorm).

And if I added the .prettierrc but didn't change tslint.json, the Prettier Reformat action would properly remove all semicolons, but then tslint would show errors everywhere.

So, this is what ended up getting the job done:

```
// .prettierrc
{ "semi": false }

// tslint.json
{ 
  "extends": ["tslint:recommended", "tslint-config-prettier"],
  "rulesDirectory": ["tslint-plugin-prettier"],
  "rules": { "prettier": [true, { "semi": false }] }
}
```
